acceptance/tests/modules/install/already_installed.rb

```diff
@@ -3,7 +3,7 @@
 extend Puppet::Acceptance::ModuleUtils
 
 hosts.each do |host|
-  skip_test "skip tests requiring forge certs on solaris and aix" if host['platform'] =~ /solaris/
+  skip_test "skip tests requiring forge certs on solaris and aix" if host['platform'] =~ /solaris|aix/
```

Comment:
Ideally this wouldn't be necessary as we should be including the ca-cert-bundle on all non-Windows platforms: https://github.com/puppetlabs/puppet-agent/blob/master/configs/projects/puppet-agent.rb#L90